docs/usage-guide/validators.md

```diff
@@ -55,7 +66,13 @@ user_answers = columbo.get_answers(interactions)
 print(user_answers)
 ```
 
-<sup>1</sup>: The regular expression for checking for an RFC 822 compliant email address is
-[overly complicated](http://www.ex-parrot.com/~pdw/Mail-RFC822-Address.html). Additionally, that only ensures that the
-text is valid. It does not confirm if the host will accept emails sent to that address or if the user is the owner of
-the email address.
+[^1]:
```

Comment:
The `footnotes` plugin seems to be generating the back link 3 times for this footnote. I think it is because the text links to this in three places. I'm not sure if there is anything to do here, it just looks weird.

Reply:
Thanks for pointing that out! I'll make the update and look into this further.